tests/test_basic_ops_dict.py

```diff
@@ -254,27 +255,40 @@ def foo(a: int, b: int) -> int:
     assert 12 == foo(**c)
 
 
-def test_items() -> None:
+def test_items_iterator_behavior() -> None:
     c = OmegaConf.create({"a": 2, "b": 10})
-    assert sorted([("a", 2), ("b", 10)]) == sorted(list(c.items()))
+    assert list(c.items()) == [("a", 2), ("b", 10)]
 
+    # This is actually not compatible with native dict:
+    # Due to implementation considerations, DictConfig items() returns a list.
+    # If that can be fixed, feel free to remove this block
     items = c.items()
     for x in [("a", 2), ("b", 10)]:
         assert x in items
 
     items2 = iter(c.items())
     assert next(items2) == ("a", 2)
     assert next(items2) == ("b", 10)
-    with pytest.raises(StopIteration):
+    with raises(StopIteration):
         next(items2)
 
 
-def test_items2() -> None:
-    c = OmegaConf.create({"a": {"v": 1}, "b": {"v": 1}})
+def test_mutate_config_via_items_iteration() -> None:
+    c = OmegaConf.create(
+        {
+            "a": {"v": 1},
+            "b": {"v": 1},
+        }
+    )
     for k, v in c.items():
         v.v = 2
 
-    assert c == {"a": {"v": 2}, "b": {"v": 2}}
+    assert c == {
+        "a": {"v": 2},
+        "b": {
+            "v": 2,
```

Comment:
Can you remove that trailing comma to have a more consistent formatting?

Reply:
It was much easier to understand what this test was doing this way.

Reply:
I will change it now that the test has a more meaningful name.

Reply:
I suspect you misunderstood which comma I was talking about, but anyway, resolving since not relevant anymore